lightbeam/send.py

```diff
@@ -78,7 +70,12 @@ def send(self):
             os.makedirs(os.path.dirname(self.lightbeam.results_file), exist_ok=True)
             
             with open(self.lightbeam.results_file, 'w') as fp:
-                fp.write(json.dumps(self.metadata, indent=4))
+                content = json.dumps(self.metadata, indent=4)
+                # failures.line_numbers are split each on their own line; here we remove those line breaks
+                def repl(m):
+                    return m.group(0).replace(' ', '').replace('\t', '').replace('\n', '')
```

Comment:
If we're already using `re.sub()` below, we may as well use it here as well to prevent three passes over the string.

`return re.sub(r"\s+", '', m.group(0))`

Reply:
Likewise, this is cleaner, good call and done.